distributed/tests/test_executor.py

```diff
@@ -3213,9 +3213,12 @@ def test_publish_simple(s, a, b):
     yield f._start()
 
     data = yield e._scatter(range(3))
-    yield e._publish_dataset(data, 'data')
-    assert 'data' in s.published_data
-    assert e.id in s.published_data['data']['clients']
+    out = yield e._publish_dataset(data, 'data')
+    assert out['success'] == 'true'
```

Comment:
The status messages are useful internally but are probably not something that the user wants to see